bluebottle/activities/states.py

```diff
@@ -164,9 +164,9 @@ def is_owner(self, user):
             cancelled,
             deleted
         ],
-        draft,
+        needs_work,
         name=_('Restore'),
-        description=_('Restore the activity. The will mark the activity as draft again.'),
+        description=_('Restore the activity. The will mark the activity as needs_work again.'),
```

Comment:
Replace `needs_work` in the description with `needs work`.

Reply:
Fair enough